insta-ui/src/app/api/generate/route.ts

```diff
@@ -0,0 +1,52 @@
+// app/api/generate/route.ts
+import { openai } from '@ai-sdk/openai';
+import { generateText } from 'ai';
+import { generateMessages } from '@/app/server/llm/prompts';
+import { GenerateRequest } from '@/app/server/llm/types';
+
+export const maxDuration = 30;
+
+export async function POST(req: Request) {
+    try {
+        const input: GenerateRequest = await req.json();
+
+        if (!input.textInput) {
+            return new Response(
+                JSON.stringify({ error: 'Text input is required' }),
+                { status: 400 }
+            );
+        }
+
+        const messages = generateMessages(
+            input.textInput,
+            input.imageInput,
+            input.previousCode
+        );
+
+        const result = await generateText({
+            model: openai('gpt-4-turbo'),
```

Comment:
Be sure to use `const` here

Reply:
It is `const` result.

Reply:
Using the const in `insta-ui/src/app/server/llm/constants.ts` would be ideal  XD